packages/linear-progress/README.md

```diff
@@ -0,0 +1,61 @@
+> ✨ Are you a part of the Material Design web community? Help us improve by filling out this <a href='https://bit.ly/materialwebsurvey'>**10 minute survey**</a>. ✨
+
+# React Linear Progress
+
+A React version of an [MDC Linear Progress](https://github.com/material-components/material-components-web/tree/master/packages/mdc-linear-progress).
```

Comment:
The _article_ (an) doesn't feel right here.

Either make it "A React version of an MDC Linear Progress **component**." or "A React version of **~an~** MDC Linear Progress"

Reply:
I think `the` is more appropriate